test/e2e_node/cpu_manager_test.go

```diff
@@ -145,6 +145,9 @@ func enableCPUManagerInKubelet(f *framework.Framework) (oldCfg *kubeletconfig.Ku
 	newCfg := oldCfg.DeepCopy()
 
 	// Enable CPU Manager using feature gate.
+	if newCfg.FeatureGates == nil {
```

Comment:
can you move this above the "Enable CPU Manager" comment? maybe right after the DeepCopy above?

Reply:
@spiffxp Thanks for review, Done. :)